docs/root/api/stats.rst

```diff
@@ -1,29 +1,32 @@
 .. _api_stats:
 
-Stats
+Pulse
```

Comment:
Would it make sense to do `Pulse (stats library)` here?

Reply:
The opening description a couple lines below states it. :)

Reply:
Yeah, the intent was to make it clear at a glance without making the reader scroll for context. Up to you though